executor/common_linux.h

```diff
@@ -1357,6 +1357,117 @@ static long syz_emit_ethernet(volatile long a0, volatile long a1, volatile long
 }
 #endif
 
+#if SYZ_EXECUTOR || __NR__syz_io_uring_put_sqes_on_ring
+//#include <linux/fs.h> // for __kernel_rwf_t
+
+// from liburing (tools/io_uring/barrier.h)
+#ifndef LIBURING_BARRIER_H
+#define LIBURING_BARRIER_H
```

Comment:
I don't think we need the include guards here.

Reply:
Fixed, we don't use that block anymore.